OJP/OJP_FareSupport.xsd

```diff
@@ -255,9 +255,9 @@
 			<xs:documentation>[related to FARE PRODUCT in TM and NeTEx] different FARE PRODUCTs that may be available with related information</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="FareProductId" type="FareProductCodeType">
+			<xs:element name="FareProductCode" type="FareProductCodeType">
```

Comment:
not sure why Code over Id ? in NeTEx the FP has an **Id** and a **PrivateCode**, but not a **Code**

Reply:
because it is a FareProductCodeType... we can harmonise it the other way.

Reply:
Transmodel has it as FareProductIdType, so I switch that part.